### PR TITLE
Add pre-introduction for Unit 4 Multimodal Models

### PR DESCRIPTION
**Pull request by snehilsanyal**
Just added a quick pre-intro for Unit 4, all three chapters. Kept in mind the latest outline posted by the groups  on the discussion section. Coming up with the introduction chapter soon 🤗 
First atomic PR for the Issue: #54 

Best,
Fusion of Text and Vision Team.
cc: @SuryaKrishna02, @johko, @snknitin

### CODE REVIEW

File: chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Exploring Multimodal Text and Vision Models: Uniting Senses in AI
+
+Welcome to the Multimodal Text and Vision Models unit! 🌐📚👁️ In this journey, we'll delve into the world where computers understand both text and images, mirroring the way we humans combine our senses to comprehend the world.
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx:3]: ```images, vidoes and text```  would be more applicable.

File: chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Exploring Multimodality 🔎🤔💭
+
+Our adventure begins with understanding why blending text and images is crucial, exploring the history of multimodal models, and discovering how self-supervised learning unlocks the power of multi-modality.
+
+1. **Fusion of Text and Vision**
+This chapter serves as a foundational exploration, enabling learners to comprehend the significance of multimodal data, its representation, and its diverse applications in various domains, laying the groundwork for deeper insights into the fusion of text and vision within AI models.
+In this chapter, you will:
+   - Acknowledge the nature of real-world multimodal data, stemming from various sensory inputs crucial for human decision-making.
+   - Explore practical implementations of multimodality in robotics, focusing on Vision Language Action models (e.g., RT2, RTX, Palm-E), showcasing their functionality and diverse applications.
```
Comment (SuryaKrishna02) [inline chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx:13]: We are also covering different domains task  and maybe having only robotics might be misleading.

File: chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx (hunk 1 above)
Comment (snehilsanyal) [reply chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx:3]: Yes sounds reasonable, thanks Surya 😄 
Will change it now.

File: chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx (hunk 2 above)
Comment (snehilsanyal) [reply chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx:13]: Sure any suggestions on which other domains to add other than robotics?

Comment (snehilsanyal) [reply chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx:13]: Thanks I will update :D

Comment (charchit7) [reply chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx:13]: Yes, agree with @SuryaKrishna02 here. Robotics will be little different.

**Review by SuryaKrishna02 — APPROVED** [review]
Perfect. LGTM

File: chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Exploring Multimodal Text and Vision Models: Uniting Senses in AI
+
+Welcome to the Multimodal Text and Vision Models unit! 🌐📚👁️ In this journey, we'll dive into the world where computers understand images, videos and text in a way how people use their senses together to understand things in the world around them. 
+
+## Exploring Multimodality 🔎🤔💭
+
+Our adventure begins with understanding why blending text and images is crucial, exploring the history of multimodal models, and discovering how self-supervised learning unlocks the power of multi-modality. The unit discusses about different modalities with a focus on text and vision. In this unit we will encounter:
```
Comment (johko) [inline chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx:7]: In the headline you are using `Multimodality` without a hyphen, so I guess you should stick to that

```suggestion
Our adventure begins with understanding why blending text and images is crucial, exploring the history of multimodal models, and discovering how self-supervised learning unlocks the power of multimodality. The unit discusses about different modalities with a focus on text and vision. In this unit we will encounter:
```

**Review by johko — APPROVED** [review]
One small suggestion, but in general LGTM :slightly_smiling_face: 

Probably you will have to iterate over the content of the other teams' sections, as they are created, but I already appreciate that you looked into their outlines and included it here :hugs:

**Review by ricard-inho — APPROVED** [review]
Looks great!

File: chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx (hunk 3 above)
Comment (snehilsanyal) [reply chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx:7]: Yes @johko I tried to get in touch with leads of other sections, as the course structure improves I will re-iterate the contents as the structure is subject to change. Thank you 🤗

Comment (snehilsanyal) [reply chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx]: Done :D

File: chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# Exploring Multimodal Text and Vision Models: Uniting Senses in AI
+
+Welcome to the Multimodal Text and Vision Models unit! 🌐📚👁️ In this journey, we'll dive into the world where computers understand images, videos and text in a way how people use their senses together to understand things in the world around them. 
+
+## Exploring Multimodality 🔎🤔💭
+
+Our adventure begins with understanding why blending text and images is crucial, exploring the history of multimodal models, and discovering how self-supervised learning unlocks the power of multimodality. The unit discusses about different modalities with a focus on text and vision. In this unit we will encounter:
+
+1. **Fusion of Text and Vision**
```
Comment (merveenoyan) [inline chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx:9]:
```suggestion
**1. Fusion of Text and Vision**
```

File: chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+This chapter serves as a foundation, enabling learners to understand the significance of multimodal data, its representation, and its diverse applications laying the groundwork for the fusion of text and vision within AI models.
+
+In this chapter, you will:
+   - Understand the nature of real-world multimodal data coming from various sensory inputs that are important for human decision-making.
+   - Explore practical applications of multimodality in robotics, search , Visual Reasoning etc., showcasing their functionality and diverse applications.
+   - Learn about diverse multimodal tasks and models focusing on Image to Text, Text to Image, VQA, Document VQA, Captioning, Visual Reasoning etc.
+   - Conclude with an introduction on Vision Language Models and cool applications including multimodal chatbots.
+
+2. **CLIP and Relatives**
```
Comment (merveenoyan) [inline chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx:18]:
```suggestion
**2. CLIP and Relatives**
```

File: chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+Moving ahead, this chapter talks about the popular CLIP model and similar vision language models. 
+In this chapter you will:
+   - Dive deep into CLIP's magic, from theory to practical applications, and explore its variations.
+   - Discover relatives like Image-bind, BLIP, and others, along with their real-world implications and challenges.
+   - Explore the functionality of CLIP, its applications in search, zero-shot classification, and generation models like DALL-E.
+   - Understand contrastive and non-contrastive losses and exploring the self-supervised learning techniques.
+
+3. **Transfer Learning: Multimodal Text and Vision**
```
Comment (merveenoyan) [inline chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx:26]:
```suggestion
**3. Transfer Learning: Multimodal Text and Vision**
```

**Review by merveenoyan — APPROVED** [review]
Thanks a lot, looks good! Just left some formatting suggestions that you can commit and we can merge after 😊

**Review by charchit7 — COMMENTED** [review]
LGTM!

File: chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx (hunk 6 above)
Comment (snehilsanyal) [reply chapters/en/Unit 4 - Mulitmodal Models/pre-intro.mdx]: Done @merveenoyan 🤗

**Pull request merged**